src/tree/FunctionTreeItem.ts

```diff
@@ -107,6 +116,40 @@ export class FunctionTreeItem extends AzureTreeItem<ISiteTreeRoot> {
 
         throw new Error(localize('keyFail', 'Failed to get key for trigger "{0}".', this._name));
     }
+
+    private async refreshTriggerUrl(): Promise<void> {
+        const triggerUrl: URL = new URL(`${this.root.client.defaultHostUrl}/api/${this._name}`);
+        const key: string | undefined = await this.getKey();
+        if (key) {
+            triggerUrl.searchParams.set('code', key);
+        }
+
+        this._triggerUrl = triggerUrl.toString();
+    }
+
+    /**
+     * https://docs.microsoft.com/azure/azure-functions/disable-function
+     */
+    private async refreshDisabledState(): Promise<void> {
+        const appSettings: WebSiteManagementModels.StringDictionary = await this.root.client.listApplicationSettings();
+        // tslint:disable-next-line:strict-boolean-expressions
+        appSettings.properties = appSettings.properties || {};
+        const projectRuntime: ProjectRuntime | undefined = convertStringToRuntime(appSettings.properties.FUNCTIONS_EXTENSION_VERSION);
+        if (projectRuntime === ProjectRuntime.v1) {
+            this._disabled = this.config.disabled;
+        } else {
+            const key: string = `AzureWebJobs.${this._name}.Disabled`;
+            /**
+             * The docs only officially mentioned 'true' and 'false', but here is what I found:
+             * The following resulted in a disabled function:
+             * true, tRue, 1
+             * The following resulted in an enabled function:
+             * false, fAlse, 0, fdsaf, 2, undefined
```

Comment:
Did somebody have a stroke while typing this?

Reply:
Lol what problem do you have with it?

Reply:
:) Exactly gotta test some edge cases - specifically a random word other than 'true' or 'false'